.hound.yml

```diff
@@ -0,0 +1,5 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
+  ignore_file: .eslintignore
+  
```

Comment:
Add a new line at the end of this file.

Reply:
I have added a new line @MuhanguziDavid.